apps/web/src/ApplicationReadyGuard.tsx

```diff
@@ -64,7 +64,7 @@ export function ApplicationReadyGuard({ children }: PropsWithChildren<{}>) {
     return <Navigate to={ROUTES.AUTH_LOGIN} replace />;
   }
 
-  if (!isOnboardingComplete() && location.pathname !== ROUTES.AUTH_APPLICATION) {
+  if (IS_UNDER_DASHBOARD !== 'true' && !isOnboardingComplete() && location.pathname !== ROUTES.AUTH_APPLICATION) {
```

Comment:
We don't want to show the onboarding page for the accounts created with the new Dashboard. cc @SokratisVidros

Reply:
Correct. Good catch, we can leave it for now although we will get back to polishing the sign-in/sign-up flow between the two apps before the first request.